services/github/github-release.service.js

```diff
@@ -85,6 +86,21 @@ class GithubRelease extends GithubAuthV3Service {
       }),
       documentation,
     },
+    {
+      title: 'GitHub release (with filter)',
+      namedParams: { user: 'RetroMusicPlayer', repo: 'RetroMusicPlayer' },
+      queryParams: {
+        sort: 'date',
+        display_name: 'release',
+        filter: '*Open Beta',
+      },
+      staticPreview: this.render({
+        version: 'Release v6.0.2 - Open Beta',
+        sort: 'date',
+        isPrerelease: false,
+      }),
+      documentation: documentation + filterDocs,
```

Comment:
The move to OpenAPI in #9014 and follow-on PRs is going to make it way easier to document services that have lots of optional query params and provide documentation relevant to individual parameters.